src/deploy/carvel/register-apps.sh

```diff
@@ -43,10 +43,13 @@ else
     # unfortunately different in docker image names and registration link.
     BROKER_NAME=rabbitmq
 fi
-
 if [ "$STREAM_APPS_RT_VERSION" = "" ]; then
-    STREAM_URI="https://dataflow.spring.io/$BROKER_NAME-${TYPE}-latest"
-elif [[ "$STREAM_APPS_RT_VERSION" = *"SNAPSHOT"* ]]; then
+    # export STREAM_APPS_RT_VERSION=2022.0.0-SNAPSHOT
+    # export STREAM_APPS_RT_VERSION=2021.1.2 # release for Boot 2.x
+    export STREAM_APPS_RT_VERSION=2024.0.0
```

Comment:
Main (`2.11.x`) should not ship w/ the 5.0 stream apps - this is for `3.0.x`

Reply:
Reverting